factory/heartbeatV2Components.go

```diff
@@ -109,6 +110,12 @@ func (hcf *heartbeatV2ComponentsFactory) Create() (*heartbeatV2Components, error
 		}
 	}
 
+	hardforkPubKey := hcf.config.Hardfork.PublicKeyToListenFrom
```

Comment:
we have the same pattern just about everywhere. Wondering if we could convert & store the hardforkPubKeyBytes in the core components

Reply:
added, great idea :pray:

Reply:
added, great idea :pray: